skins/oasis/css/core/ads-porvata.scss

```diff
@@ -0,0 +1,282 @@
+@import '../../../../extensions/wikia/ArticleVideo/styles/video-shared-variables';
+
+$wikia-top-ads-index: 2;
+
+$default-progress-bar-color: #b7b7b7;
+$default-progress-bar-current-time-color: #666;
+$outstream-progress-bar-color: #666;
+$outstream-progress-bar-current-time-color: #b7b7b7;
+$video-player-button-size: 24px;
+
+$floating-ad-margin: 20px;
+$floating-ad-index: $wikia-top-ads-index + 10;
+$floating-ad-video-width: 320px;
+$floating-ad-animation-speed: $video-collapse-animation-speed;
+$floating-ad-featured-video-gap: $video-container-width;
+
+$vpaid-z-index: 10;
+$video-interactive-area-z-index: $vpaid-z-index + 1;
+
+@mixin absolute-fill() {
+	height: 100%;
+	left: 0;
+	position: absolute;
+	width: 100%;
+	top: 0;
+}
+
+.wikia-ad {
+	.video-overlay {
+		@include absolute-fill();
+	}
+
+	.video-display-wrapper {
+		background: #000;
+		height: 100%;
+		position: relative;
+		width: 100%;
+
+		video, &.vpaid-enabled > div {
+			@include absolute-fill();
+		}
+
+		&.vpaid-enabled video {
+			z-index: $vpaid-z-index;
+		}
+
+		.video-player {
+			background: #000;
+			margin: 0 auto;
+			pointer-events: auto;
+		}
+	}
+
+	.replay-overlay {
+		background: rgba(0,0,0,0.3) url('/extensions/wikia/AdEngine/js/video/player/ui/images/play_button.svg') center/8% no-repeat;
+		cursor: pointer;
+		height: 100%;
+		position: absolute;
+		z-index: 1;
+	}
+}
+
+.video-player {
+	position: relative;
+
+	&#{&}-left, &#{&}-right {
+		z-index: $wikia-top-ads-index;
+	}
+
+	&#{&}-right {
+		right: 0;
+	}
+
+	video {
+		left: 0;
+	}
+
+	.progress-bar {
+		background: $default-progress-bar-color;
+		bottom: 0;
+		height: 8px;
+		left: 0;
+		position: absolute;
+		width: 100%;
+	}
+
+	.current-time {
+		@include transition(width, 1s, linear, 0);
+		background: $default-progress-bar-current-time-color;
+		height: 100%;
+		max-width: 100%;
+		width: 0;
+	}
+
+	.pause-overlay {
+		cursor: pointer;
+		height: 100%;
+		position: absolute;
+		top: 0;
+		width: 100%;
+	}
+
+	.close-ad {
+		background-image: url('/extensions/wikia/AdEngine/js/video/player/ui/images/close_button.svg');
+		cursor: pointer;
+		height: $video-player-button-size;
+		margin: 10px 10px 30px 30px;
+		position: absolute;
+		right: 0;
+		top: 0;
+		width: $video-player-button-size;
+	}
+
+	/* TODO remove .ima-mute-div and all references in next releases (keep .volume-control only) */
```

Comment:
Do we have a ticket for that? Can we add it here?

Reply:
No, there is no ticket for that and I'm not sure if it's worth to put it on board. Next time this file will be touched Sonar will remind us :)